### ISSUE TITLE
Add .data.R files for knitr models

### ISSUE THREAD

**Issue by seantalts** [stan-dev/example-models#151]
It would be really useful to be able to run these models programmatically as part of a regression and performance test suite, but for that we need to get the data out of the notebook and into a file.

**Comment by mitzimorris** [comment]
agreed that some of the models in these case studies could be useful for regression and performance testing, however many of the models in these case studies make points in service of the case study and are not really appropriate.

I've removed all of the "garbage-auto-generated.data.R" files which were added in June - these are distracting to consumers of the case studies.  furthermore, the authors of these case studies should have control over the contents of their case study.

I suggest closing this issue.
